Engine/AI/NavMesh.h

```diff
@@ -14,7 +14,7 @@
 class PanelNavMesh;
 class dtNavMeshQuery;
 class dtNavMesh;
-class dtMeshTile;
+struct dtMeshTile;
```

Comment:
mmm cs here?

Reply:
Why code style? :D

Reply:
I asked that to know if we should separate between classes and structs like we do when using includes from our code or from other libraries.

Also after `struct dtMeshTile ` there is a `class DetourNavMesh` that if we put it alphabetically should be the first one, then the other classes ref.

Reply:
Yeah, we should. So Im changing it!